ClimsoftVer4/ClimsoftVer4/formMetadata.vb

```diff
@@ -1751,4 +1759,35 @@ Err:
         End If
     End Sub
 
+    Private Sub CombEW_SelectedIndexChanged(sender As Object, e As EventArgs) Handles CombEW.SelectedIndexChanged
+        Dim myLon As Double
+        If CombEW.SelectedItem = "W" Then
+
+            myLon = (-1) * (txtDegreesLon.Text + txtMinutesLon.Text / 60 + txtSecondsLon.Text / 3600)
```

Comment:
@isedwards - This is what I meant:
 txtDegreesLat.Text = 0
 txtDegreesLon.Text = 0
 txtMinutesLat.Text = 0
 txtMinutesLon.Text = 0
 txtSecondsLat.Text = 0
 txtSecondsLon.Text = 0
 CombNS.SelectedItem = "N"
CombEW.SelectedItem = "E"
If I remove 0s ie:txtDegreesLat.Text ="" I get an error message.